src/MetaMorpho.sol

```diff
@@ -635,6 +631,20 @@ contract MetaMorpho is ERC4626, ERC20Permit, Ownable2Step, Multicall, IMetaMorph
         assets -= _simulateWithdrawMorpho(assets);
     }
 
+    /// @dev Returns the maximum amount of assets that the vault can supply on Morpho.
+    function _maxDeposit() internal view returns (uint256 totalSuppliable) {
```

Comment:
Could have been written in maxDeposit() directly, but not mandatory

Reply:
not really because `maxDeposit` is waiting for an `address` so I think it's cleaner like that

Reply:
And it's used twice!